### PR TITLE
9525 total users in builder

### LINKED ISSUE #9525: Provide the number of organization users in the dataset and builder pages
In order to avoid an unnecessary request to `users` endpoint.
Changes in frontend and backend, I feel. 💋

### PR DESCRIPTION
**Pull request by javitonino**
Closes #9525 

Finally passing the data though the organization model, looked like a cleaner way to do it and can be reused in the future.

### CODE REVIEW

**Comment by javitonino** [comment]
CR @nobuti @gfiorav

**Comment by nobuti** [comment]
\o/

File: lib/assets/javascripts/cartodb3/components/modals/publish/share-with-view.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ module.exports = CoreView.extend({
   _getPeople: function () {
     var types = this._acl.pluck('type');
     if (types.indexOf('org') > -1) {
-      return this._userModel.usersOrganization.total_user_entries;
+      return this._userModel._organizationModel.get('user_count') - 1;
```
Comment (nobuti) [inline lib/assets/javascripts/cartodb3/components/modals/publish/share-with-view.js]: You can use too `this._userModel.getOrganization()` to get `_organizationModel`

**Comment by nobuti** [comment]
LSGTM

File: lib/assets/javascripts/cartodb3/components/modals/publish/share-with-view.js (hunk 1 above)
Comment (javitonino) [inline lib/assets/javascripts/cartodb3/components/modals/publish/share-with-view.js]: Much nicer than using a "private" `_` prefixed variable. Changing.

**Comment by gfiorav** [comment]
Do it!

**Pull request merged**